app/main.py

```diff
@@ -1,3 +1,59 @@
+from __future__ import annotations
+from typing import Any, Union
+
+
+OtherInputType = Union[float, int]
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: OtherInputType) -> Distance:
+        return Distance(
+            self.km + self.number_return(other)
+        )
```

Comment:
`other` also can be an instance of `Distance` class. You must show it in type hints